### PR TITLE
BZ2073611 - Fixing step for installing on single node

### PR DESCRIPTION
**Pull request by kelbrown20**
For versions 4.9+ 
[Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2073611)

Ready for QE: @gpei 

Preview: [Installing OpenShift on a single node
](https://deploy-preview-44529--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno)

### CODE REVIEW

File: modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ All DNS records must be subdomains of this base domain and include the cluster n
 
 . Select *Install single node OpenShift (SNO)*.
 
-. Read the limitations for installing {product-title} on a single node.
+. Read {product-version} Release notes, which outlines some of the limitations for installing {product-title} on a single node.
```
Comment (mjpytlak) [inline modules/install-sno-generating-the-discovery-iso-with-the-assisted-installer.adoc:33]:
```suggestion
. Read the {product-version} Release notes, which outline some of the limitations for installing {product-title} on a single node.
```

**Comment by mjpytlak** [comment]
@kelbrown20 One small comment.

**Comment by gpei** [comment]
LGTM

**Review by adellape — APPROVED** [review]

**Pull request merged**